### ISSUE TITLE
GSDX: Dragon Ball Z Budokai Tenkaichi 3 Graphic Bug.

### ISSUE THREAD

**Issue by woj1993** [PCSX2/pcsx2#2001]
PCSX2 version:
v1.5.0-dev-2136-gbab9826

PCSX2 options:
Safest . MTVU off.

Plugins used:
GSDX, Lilypad, SPU2-X, Rest Default null plugins.

GSDX configuration:
OpenGL
Accurate Date
Blending High (Tried Ultra. Doesn't meter)
CRCHack Auto
16x anisotropic filtering. (Tried off. Doesn't meter)
Mipmaping full. (Tried off. Doesn't meter)

Description of the issue:
Character too close to screen should be transparent blue. Transparent blue color is in random place on screen in battle mode all the time.
How it looks:
![gsdx_20170705125326](https://user-images.githubusercontent.com/11420970/27861748-69a514ce-6182-11e7-86c3-5b3b583f34b9.png)
How it should:
![gsdx_20170705125334](https://user-images.githubusercontent.com/11420970/27861761-79363b20-6182-11e7-9b12-baa12574a137.png)

How to reproduce the issue:
Battle on a level. (Preferably with sunlight because it is more visible).

PC specifications:
CPU: Intel Core I5 2300 GPU: MSI GT440 OS: Windows 10 64 Bit

Dump: [Dragon ball Z Budokai Tenkaichi 3 GSDX bug.zip](https://github.com/PCSX2/pcsx2/files/1124416/Dragon.ball.Z.Budokai.Tenkaichi.3.GSDX.bug.zip)

**Comment by lightningterror** [comment]
Hmm you mentioned this here as well https://github.com/PCSX2/pcsx2/pull/1900#issuecomment-307755629

I'm not able to replicate the issue , for me the effect is skipped in Direct3D and OpenGL even with no crc.

Can you show a pic how it looks on Direct3D

Also can you tell me your game CRC number , serial , region.

**Comment by woj1993** [comment]
Direct3D11: 
![gsdx_20170705145222](https://user-images.githubusercontent.com/11420970/27865121-a6af9e8e-6191-11e7-8566-17d9fecbe86d.png)
Region:PAL
CRC: A422BB13
Serial: SLES-54945

**Comment by lightningterror** [comment]
@ssakash do you think this could be related to what you said here https://github.com/PCSX2/pcsx2/pull/1900#discussion_r120854550

**Comment by woj1993** [comment]
@lightningterror You mean this 
> if(fi.FBP == 0x02e00) 
 			{ 
				skip = 3; // NTSC 

I don't say it only sshakas. Tell this. This code will work if I disable crcHack? If not it is not related.I wold say it looks more like this hack "Sky texture (depth related). On Direct3D the blue sky texture is shown on the whole screen in front.".

**Comment by lightningterror** [comment]
I was talking about GetVideoMode())
Maybe there are crc conflicts.

**Comment by woj1993** [comment]
Ok. Thanks.

**Comment by lightningterror** [comment]
Can you try this out?
https://drive.google.com/open?id=0B8cQAq-Ko9-uLVJlQzU5eTVxb0E

Both dx11 and opengl. Default (automatic crc)
The high brightness/contrast issue should also be gone.

The effects aren't fixed. I just configured the crc hacks that skip effects a little.

**Comment by woj1993** [comment]
@lightningterror Hi.
Thank you for "fixing" it.
How it looks:
OpenGL HW.
![gsdx_20170721122749](https://user-images.githubusercontent.com/11420970/28460254-89c7506e-6e11-11e7-8c5b-c0e838a5983b.png)
OpenGL SW.
![gsdx_20170721122802](https://user-images.githubusercontent.com/11420970/28460259-8e08d44a-6e11-11e7-919b-5aab2950a6c7.png)
DX11HW
![gsdx_20170721122831](https://user-images.githubusercontent.com/11420970/28460260-91a6bdf6-6e11-11e7-981d-f6d1f57250f1.png)
I think we lowered brightness/contrast too much. Look at the sky in SW mode and in HW.
Or if this is error in SW mode we should report it to Gregory.

P.S. This test was only 1 stage I didn't test it fully. Should I test any other games?

**Comment by lightningterror** [comment]
They should look like that , pcsx2 has a bit lower brightness.
You can increase the brightness in the shader configuration.

**Comment by woj1993** [comment]
So it is error in software rendering?
I have blending set to high. I tested only this stage at night too. I was wondering if other games can be affected by this? I will edit this comment when I test other stages.

**Comment by lightningterror** [comment]
I asked about blending because character outlines looked kinda thin but maybe that's just me.

**Comment by MrCK1** [comment]
@lightningterror Is this still broken?

**Comment by lightningterror** [comment]
It is. Both pal and ntsc are missing the character texture on ogl and dx.

The other issue that is skipped with a crc hack is a depth effect.

**Comment by Stickend** [comment]
I don't know exactly in what build did it wrong, but this issue is present again. I remember this was fixed time ago, but now it came back again.

All with normal settings, nothing changed on my emulator, using latest version at the time of this post.

**Comment by Stickend** [comment]
@lightningterror Redownloaded latest version again separetly to test again. Didn't change anything on settings, yet the issue is still present. I don't know why.

![image](https://user-images.githubusercontent.com/34516283/221728975-f0716efd-d215-4c00-8838-94e93fb1ccc1.png)

**Comment by JordanTheToaster** [comment]
> @lightningterror Redownloaded latest version again separetly to test again. Didn't change anything on settings, yet the issue is still present. I don't know why.
> 
> ![image](https://user-images.githubusercontent.com/34516283/221728975-f0716efd-d215-4c00-8838-94e93fb1ccc1.png)

Are you using a modded game?

**Comment by Stickend** [comment]
This is the last version I used this wasn't an issue for me. That version was, at least, 1.7.2366

![image](https://user-images.githubusercontent.com/34516283/221729862-dbe19696-8f9f-4095-a986-5a6837de8d3f.png)

**Comment by Stickend** [comment]
I'm using a modded version, yet this wasn't a problem ever, until now for some reason, as you can see in my previous post.

**Comment by JordanTheToaster** [comment]
> I'm using a modded version, yet this wasn't a problem ever, until now for some reason, as you can see in my previous post.

Please use an unmodded game before making issues. Mods are not our responsibility.

**Comment by Stickend** [comment]
> > I'm using a modded version, yet this wasn't a problem ever, until now for some reason, as you can see in my previous post.
> 
> Please use an unmodded game before making issues. Mods are not our responsibility.

It's quite strange, there was never an issue before. I remember opening a post explaining one of the most bugged effects in this game using indeed a modded version. That issue was resolved time ago too, and the information I gave was based on the modded version. That's why it makes me wonder what happened now (that makes it different when it didn't before).

https://github.com/PCSX2/pcsx2/issues/2219
